web/packages/teleport/src/stores/storeUserContext.ts

```diff
@@ -157,14 +157,22 @@ export default class StoreUserContext extends Store<UserContext> {
   // has access to download either teleport binaries or the license.
   // Since the page is used to download both of them, having access to one
   // is enough to show access this page.
-  // This page is only available for `dashboards`.
+  // This page is only available for `dashboards` and cloud customers.
   hasDownloadCenterListAccess() {
     return (
-      cfg.isDashboard &&
-      (this.state.acl.license.read || this.state.acl.download.list)
+      cfg.isCloud ||
+      (cfg.isDashboard &&
+        (this.state.acl.license.read || this.state.acl.download.list))
     );
   }
 
+  // hasSupportPageLinkAccess checks if the user
+  // has access to a Support external link in the side menu.
```

Comment:
@avatus is in the middle of removing the side bar, was the dashboard view discussed with this refactor?

Reply:
This change is now in master btw: https://github.com/gravitational/teleport/pull/36368

You may need to update how this is displayed @mcbattirola

Reply:
Thanks for pointing that out, I wasn't aware of the new topbar.

I've talked with Kenny, we'll be changing some stuff around:

- On dashboards, instead of `Resources`, the first item in the top nav will be a `Downloads` tab.
- On Cloud, we'll move the link to another place (probably under the user menu, but I'm awaiting confirmation on that).

Since this PR doesn't introduce the top bar link, only changes the show condition, I think we can merge this and implement the changes listed above in a new PR.